common/src/main/proto/TransportMessages.proto

```diff
@@ -98,6 +98,7 @@ enum MessageType {
   REGISTER_APPLICATION_RESPONSE = 75;
   WORKER_EVENT_REQUEST = 76;
   WORKER_EVENT_RESPONSE = 77;
+  APPLICATION_META = 78;
```

Comment:
```suggestion
  SAVE_APPLICATION_META = 78;
```

Reply:
Could you please explain why should we change the message type to `SAVE_APPLICATION_META`? The message contains application meta information so I think type should be `APPLICATION_META`.

Reply:
@otterc, IMO, this behavior of request is that saves the application meta, therefore it's better to adding the operation in this message type.

Reply:
I think this message encapsulates Application meta and the type should be called `APPLICATION_META`. In a future PR, the workers will have the capability to request Master for the application meta which will be called `ApplicationMetaRequest` and then the master responds back with `ApplicationMeta`. IMO, I don't think we should call it `SAVE_APPLICATION_META` here.